Tests/NIOHTTPCompressionTests/HTTPRequestDecompressorTest.swift

```diff
@@ -63,14 +63,10 @@ class HTTPRequestDecompressorTest: XCTestCase {
         let buffer = ByteBuffer.of(bytes: [120, 156, 75, 76, 28, 5, 200, 0, 0, 248, 66, 103, 17])
         let compressed = compress(buffer, "gzip")
 
-        do {
-            try channel.writeInbound(HTTPServerRequestPart.body(compressed))
-        } catch let error as NIOHTTPDecompression.DecompressionError {
-            switch error {
-            case .limit:
-                // ok
-                break
-            default:
+        XCTAssertThrowsError(try channel.writeInbound(HTTPServerRequestPart.body(compressed))) {error in
+           if case .limit = error as? NIOHTTPDecompression.DecompressionError {
```

Comment:
For Swift 5.0 these all need to be `if case .some(.limit)`

Reply:
Thanks, I did the change